src/nxdoc/nuxeo-server/installation/compatibility-matrix.md

```diff
@@ -123,7 +123,9 @@ Nuxeo is shipped with an embedded Tomcat server.
 
 ## OpenSearch
 
-Nuxeo Platform is tested against an OpenSearch cluster {{! multiexcerpt name='opensearch_cluster_supported_version'}}1.3.19{{! /multiexcerpt}} using OpenSearch Java libraries {{! multiexcerpt name='opensearch_libraries_supported_version'}}1.3.19{{! /multiexcerpt}}.
+Nuxeo Platform is tested against 2 OpenSearch clusters
+- an OpenSearch cluster {{! multiexcerpt name='opensearch_cluster_supported_version'}}1.3.20{{! /multiexcerpt}} using OpenSearch Java libraries {{! multiexcerpt name='opensearch_libraries_supported_version'::}}1.3.20{{! /multiexcerpt}}.
+- an OpenSearch cluster {{! multiexcerpt name='opensearch2_cluster_supported_version'}}2.18.0{{! /multiexcerpt}} using OpenSearch Java libraries {{! multiexcerpt name='opensearch_libraries_supported_version'::}}2.23.0{{! /multiexcerpt}}.
```

Comment:
I think this is not completely true, we do not test Nuxeo Platform against OpenSearch 2.x.